manifests/celestial-strands

```diff
@@ -0,0 +1,13 @@
+name: Celestial Strands
+authors: Saugia
+homepage: https://github.com/Saugia/celestial-strands
+license: GPL-3.0-or-later
```

Comment:
You use the AGPL (GNU "Affero" GPL)

```suggestion
license: AGPL-3.0-or-later
```

Reply:
I swapped my LICENSE file to GPL, as that is what I intended. Does that work?

Reply:
yep